src/types/unity.ts

```diff
@@ -0,0 +1,119 @@
+import {RemoteDataState} from '@influxdata/clockface'
+import {User as GenUser, Invite as GenInvite} from 'src/client/unityRoutes'
+
+export type Role = 'owner' | 'member'
+
+export interface CloudUser extends GenUser {
+  status: RemoteDataState
+}
+export interface Invite extends GenInvite {
+  status: RemoteDataState
+}
+
+export type DraftInvite = Omit<GenInvite, 'id' | 'status' | 'expiresAt'>
+
+export interface InviteErrors {
+  email?: string[]
+}
+
+export interface InviteErrorsJSON {
+  errors: InviteErrors
+}
+
+export type GetOrgsUsersResult =
+  | GetOrgsUsersOKResult
```

Comment:
Oh, cool! Does TS allow a leading `|` here with nothing before it?  That's a nice trailing-comma-like feature.